### ISSUE TITLE
Resuming model from model.loss.best doesn't work

### ISSUE THREAD

**Issue by siddalmia** [espnet/espnet#2359]
Hi,

When I try to resume my model training / fine-tune my pretrained model from model.loss.best it errors as it is unable to find "trainer" in snapshot_dict. 

Is there a way to override this?

Thanks
Sid

**Comment by siddalmia** [comment]
Ok looks like I can use enc-init and dec-init. 

Thanks and sorry for the inconvenience.
